libsolidity/inlineasm/AsmScopeFiller.cpp

```diff
@@ -106,6 +106,12 @@ bool ScopeFiller::operator()(Switch const& _switch)
 	return success;
 }
 
+bool ScopeFiller::operator()(ForLoop const& _forLoop)
+{
+	/// FIXME: implement
```

Comment:
Because it is not at all useful? I don't think it should take that long to finish the analyzer on this if we dedicate some time to it :)